Server/src/migrations/SeedDatabase.ts

```diff
@@ -341,6 +344,9 @@ export class SeedDatabase1611344612000 implements MigrationInterface {
   }
 
   public async down(queryRunner: QueryRunner): Promise<any> {
+    // Temporary line, will be removed on merge
+    await queryRunner.query('ALTER TABLE graphstate RENAME TO savedgraphs');

```

Comment:
as an fyi, you can write the comment as such: //todo .... stuff to do
you can then have extensions installed in your ide to let you know where you have left todos
here is my example:
![image](https://user-images.githubusercontent.com/22376976/105712813-57006780-5ee8-11eb-899c-daa8a16869e0.png)
it links me right to the place where i left the comment

Reply:
This is solid advice, I didn't know this was a thing. What extension does this notice? Want to grab it myself!

Reply:
![image](https://user-images.githubusercontent.com/22376976/105797774-9bc6e580-5f5e-11eb-9289-69432b2e1fd7.png)